### PR TITLE
Move Classes into Virtfs namespace

### PR DESCRIPTION
**Pull request by movitto**

### CODE REVIEW

File: lib/virtfs/iso9660/Iso9660.rb [hunk 1]
```diff
@@ -0,0 +1,293 @@
+require 'rufus/lru'
+
+# Iso9660 stuff.
+require 'fs/iso9660/boot_sector'
+require 'fs/iso9660/directory_entry'
+require 'fs/iso9660/directory'
+require 'fs/iso9660/file_data'
+
+module Iso9660
```
Comment (roliveri) [inline lib/virtfs/iso9660/Iso9660.rb:9]: Shouldn't this module be defined under the `module VirtFS` namespace?

File: lib/virtfs/iso9660/Iso9660Probe.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module Iso9660Probe
```
Comment (roliveri) [inline lib/virtfs/iso9660/Iso9660Probe.rb:1]: Namespace here to, should this be: `VirtFS::Iso9660::Iso9660Probe`?

File: lib/virtfs/iso9660/boot_sector.rb [hunk 3]
```diff
@@ -0,0 +1,167 @@
+require 'sys-uname'
+
+module VirtFS::ISO9660
```
Comment (roliveri) [inline lib/virtfs/iso9660/boot_sector.rb:3]: Hmmm, how does this module relate to the `Iso9660` module I commented on earlier?
Is one for the implementation and the other for the wrapper?

Comment (roliveri) [inline lib/virtfs/iso9660/boot_sector.rb:3]: @movitto

**Comment by movitto** [comment]
@roliveri the changes you commented on were from the 2nd commit in the set which corresponded to the 2nd PR in the series (PR #3) which you merged. This PR series consists of a series of sequential changes importing the ISO9660 module from manageiq and adapting it to the virtfs interface.

While I did my best to isolate each patch/PR to atomic changes, some are "transitionary" and with the switchover encompassing more than one patch. There are just natural fall-outs of the conversion / porting process. If anything else needs explaining, feel free to ping.

**Comment by movitto** [comment]
@roliveri those two modules specifically are deleted later on in the series after all their functionality has been extracted before that:

https://github.com/ManageIQ/virtfs-iso9660/pull/9/commits/2e277b88f435d4edb95eae8e48cfa61e80cf0d58

**Comment by movitto** [comment]
@roliveri the PR set has been rebased against the current master

**Pull request merged**